.devcontainer/default/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM swiftlang/swift:nightly-main-jammy
+
+RUN \
+  #
+  # Disable apt interactive prompts for this RUN command
+  export DEBIAN_FRONTEND="noninteractive" && \
+    # 
+    # Update apt package list
+    apt-get update && \
+    # 
+    # Install sourcekit-lsp dependencies
+    apt-get install -y \
+      libsqlite3-dev libncurses5-dev python3
```

A little nitpicky but I think it would be more readable with a little fewer newlines. And flushing it to the left seems to match VS Code’s formatter for Dockerfiles.

```suggestion
RUN \
  # Disable apt interactive prompts for this RUN command
  export DEBIAN_FRONTEND="noninteractive" && \
  # Update apt package list
  apt-get update && \
  # Install sourcekit-lsp dependencies
  apt-get install -y libsqlite3-dev libncurses5-dev python3
```